secrets/helper/helper.go

```diff
@@ -79,22 +79,37 @@ func SetupGCPSSM(
 	)
 }
 
-func InitValidatorKey(secretsManager secrets.SecretsManager) (*ecdsa.PrivateKey, error) {
-	// Generate the IBFT validator private key
-	validatorKey, validatorKeyEncoded, keyErr := crypto.GenerateAndEncodePrivateKey()
-	if keyErr != nil {
-		return nil, keyErr
+func InitValidatorKey(secretsManager secrets.SecretsManager) (types.Address, error) {
+	validatorKey, validatorKeyEncoded, err := crypto.GenerateAndEncodePrivateKey()
+	if err != nil {
+		return types.ZeroAddress, err
 	}
 
+	address := crypto.PubKeyToAddress(&validatorKey.PublicKey)
+
 	// Write the validator private key to the secrets manager storage
 	if setErr := secretsManager.SetSecret(
 		secrets.ValidatorKey,
 		validatorKeyEncoded,
 	); setErr != nil {
-		return nil, setErr
+		return types.ZeroAddress, setErr
+	}
+
+	return address, nil
+}
+
+func GetBLSPubkeyFromValidatorKey(secretsManager secrets.SecretsManager) ([]byte, error) {
+	ecdsaKey, err := crypto.ReadConsensusKey(secretsManager)
+	if err != nil {
+		return nil, err
+	}
+
+	pubkeyBytes, err := crypto.ECDSAToBLSPubkey(ecdsaKey)
+	if err != nil {
+		return nil, err
```

Comment:
Why is this in the `secrets` package, when it only contains crypto functionality?

Reply:
Moved in b12f4bdc7253b1bbb514a566e2e1bb25b2602b1d